kernel/main.c

```diff
@@ -200,10 +202,13 @@ read_multiboot_info(void)
       const char *name = TO_PTR(mbi->boot_loader_name);
 
       if (!strcmp(name, "TILCK_EFI")) {
+         struct tilck_extra_boot_info *extra_boot_info = TO_PTR(mbi->apm_table);
 
          printk("Multiboot: detected the TILCK_EFI bootloader\n");
-         printk("Multiboot: ACPI root ptr: %p\n", TO_PTR(mbi->apm_table));
-         acpi_set_root_pointer(mbi->apm_table);
+         printk("Multiboot: ACPI root ptr: %p\n", TO_PTR(extra_boot_info->RSDP));
```

Comment:
Missing `printk()` about the RT pointer, as requested in the previous review. Add something like:
```
printk("Multiboot: UEFI RT ptr: %p\n", TO_PTR(extra_boot_info->RT));
```